Source/WebInspectorUI/UserInterface/Views/ComputedStyleSection.js

```diff
@@ -25,8 +25,10 @@
 
 WI.ComputedStyleSection = class ComputedStyleSection extends WI.View
 {
-    constructor(delegate)
+    constructor(delegate, {variablesGroupType} = {})
     {
+        console.assert(!variablesGroupType || Object.values(WI.CSSStyleDeclaration.VariablesGroupType).includes(variablesGroupType), "Unknown variables group type.");
```

Comment:
please just log the `variablesGroupType` instead of having a message

```
console.assert(!variablesGroupType || Object.values(WI.CSSStyleDeclaration.VariablesGroupType).includes(variablesGroupType), variablesGroupType);
```
anyone reading this code will immediately be able to understand what the problem is, and knowing _what_ the invalid value is is often more useful than generally knowing that there is an invalid value